VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -92,21 +92,22 @@ context('VATextInput', () => {
   describe('when there is helper text', () => {
     it('should display it', async () => {
       initializeTestInstance('email', '', 'back.a11yHint')
-      expect(testInstance.findAllByType(TextView)[1].props.children).toEqual('Navigates to the previous page')
+      expect(screen.getByText('Navigates to the previous page')).toBeTruthy()
+      expect(testInstance.findByProps({ 'children': 'Navigates to the previous page' })).toBeTruthy()
     })
   })
 
   describe('when there is an error', () => {
     it('should display it', async () => {
       initializeTestInstance('email', '', '', 'ERROR')
-      const allTextViews = testInstance.findAllByType(TextView)
-      expect(allTextViews[allTextViews.length - 1].props.children).toEqual('ERROR')
+      expect(screen.getByText('ERROR')).toBeTruthy()
+      expect(testInstance.findByProps({ 'children': 'ERROR' })).toBeTruthy()
     })
 
     it('should set the border color to error and make the border thicker', async () => {
       initializeTestInstance('email', '', '', 'ERROR')
-      expect(testInstance.findAllByType(Box)[4].props.borderColor).toEqual('error')
-      expect(testInstance.findAllByType(Box)[4].props.borderWidth).toEqual(2)
+      expect(testInstance.findByProps({ 'borderColor': 'error' })).toBeTruthy()
```

Comment:
remove the test instance test and update it to screen.getBy tests

Reply:
Updated to use `screen.getByText`